--- MG5_aMC_v3_3_2/madgraph/interface/madgraph_interface.py	2022-03-22 06:23:22.000000000 -0300
+++ MG5_aMC_v3_3_2_mod/madgraph/interface/madgraph_interface.py	2024-05-30 15:44:10.762503514 -0300
@@ -3025,8 +3025,8 @@
         """Initializing before starting the main loop"""
 
         self.prompt = 'MG5_aMC>'
-        if madgraph.ReadWrite: # prevent on read-only disk
-            self.do_install('update --mode=mg5_start')
+        # if madgraph.ReadWrite: # prevent on read-only disk
+        #     self.do_install('update --mode=mg5_start')
 
         # By default, load the UFO Standard Model
         logger.info("Loading default model: sm")
@@ -3104,8 +3104,8 @@
             os.remove(pjoin(self._done_export[0],'RunWeb'))
 
         value = super(MadGraphCmd, self).do_quit(line)
-        if madgraph.ReadWrite: #prevent to run on Read Only disk
-            self.do_install('update --mode=mg5_end')
+        # if madgraph.ReadWrite: #prevent to run on Read Only disk
+        #     self.do_install('update --mode=mg5_end')
         misc.EasterEgg('quit')
         
         
